tests/auto/mocks/qmozsecurity/qmozsecurity.h

```diff
@@ -0,0 +1,33 @@
+/****************************************************************************
+**
+** Copyright (C) 2015-2019 Jolla Ltd.
+** Copyright (c) 2019 Open Mobile Platform LLC.
+**
+** Contact: Raine Makelainen <raine.makelainen@jollamobile.com>
```

Comment:
Drop these contacts... 

Personally I prefer following style but it's better to align the rest of the project.
```
/*
 * Copyright (C) 2015 - 2019 Jolla Ltd.
 * Copyright (c) 2019 Open Mobile Platform LLC.
 */
```

Reply:
I've tried to align with the rest of the project, but the copyright and licence headers use different styles in most headers anyway, so if you want me to change to your preferred style I'm not sure it would cause a terrible inconsistency.

Reply:
If you would align just the files that you're changing... I can create another PR for the rest to align with copyright + license on the simple (sample) comment block... 

```
/* 
 * Copyright (c) 2019 Open Mobile Platform LLC.
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/.
 */
```

Reply:
All the files here should now align with this.